site/content/en/docs/user-manual/supported_formats.md

```diff
@@ -118,6 +118,9 @@ List of supported formats:
   - [Format specification](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/htmldoc/index.html)
   - [Dataset example](https://github.com/openvinotoolkit/datumaro/tree/develop/tests/assets/voc_dataset)
   - [Format documentation](/docs/formats/pascal_voc)
+- Super Resolution
```

Comment:
Shouldn't it be prefixed Common as well?

Reply:
I'm not sure about this. For WB it is, but in AC it is `super_resolution_converter`.

Reply:
As I see, in AC there are 2 kinds of this format, and this one looks more like the directory-based variant.